frontend/src/core/dashboard/views/dashboardView.js

```diff
@@ -108,7 +108,7 @@ define(function(require){
         title: 1
       };
 
-      this.setUserPreference('sort','asc');
+      this.setUserPreference('sort','asc', true);
```

Comment:
Why is this data only stored in the session?

Reply:
I believe my thinking here was that it might be confusing to auto-apply filters and sorts across sessions, as the user will likely have forgotten what's turned on. 

This is less of an issue with the sorts though, as nothings hidden

Reply:
For me, i would like sorts to be stored across sessions as I find I'm constantly swapping to recent. I think any that have ui elements should be saved as its obvious they are set.

Reply:
Coolio, makes sense to me 👍